FirebaseAuth/Sources/Swift/Backend/RPC/GetOOBConfirmationCodeRequest.swift

```diff
@@ -78,6 +78,9 @@ private let kCanHandleCodeInAppKey = "canHandleCodeInApp"
 /// The key for the "dynamic link domain" value in the request.
 private let kDynamicLinkDomainKey = "dynamicLinkDomain"
```

Comment:
Note: removed the planned deprecation annotation `@deprecated` due to build failed.  At time we are launching, for a period of time we will accept both dynamicLinkDomain and LinkDomain at the same time and GCIP will pick the one according to the Admin API settings.  After the shutdown onboarding phase, we will add back the annotation.